internal/adapters/grpc/server.go

```diff
@@ -1,27 +1,32 @@
+// Package grpc defines gRPC server
```

Comment:
@sepehr-dh99 move grpc package to `api/grpc` additional proto and generated proto.